docs/use_case/postv1.md

```diff
@@ -0,0 +1,77 @@
+This initial version of the demo #post app will allow you to create and visualise your own post(s). It will also showcase examples of the ONEM's python SDK usage.
+
+So let's navigate to the **postv1** folder and create/activate our virtual environment:
+
+```bash
+onem@local:~$ cd ~/postv1 ; pipenv shell --python 3.7
+```
+
+
+Also please make sure that:
+
+ - the database is created (even if we will store the posts in-memory).
```

Comment:
There is no specific mention of database configuration in the run locally tutorial

Reply:
correct, I'll add it